### PR TITLE
Added Gigamon GigaVUE basic support

### PR DESCRIPTION
**Pull request by hanserasmus**
Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @hanserasmus 
Thanx for your contribution. Could you change your discovery to match the new syntax, like in Exa : 
LibreNMS/OS/Exa.php 

The goal is to move some tasks (Serial, hardware, version etc) from the poller to the discovery process.

Thanx

**Comment by hanserasmus** [comment]
Big shout-out and a lot of credit needs to go to @SourceDoctor and @TheGreatDoc for helping me through this. Thank you!

File: LibreNMS/OS/Gigamon.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2020 Tony Murray
```
Comment (TheGreatDoc) [inline LibreNMS/OS/Gigamon.php:22]: Change this to yours

**Comment by TheGreatDoc** [comment]
Also, note you have uploaded 2 snmprec and 2 json, probably due to the file/os mismatch

File: LibreNMS/OS/Gigamon.php (hunk 1 above)
Comment (hanserasmus) [reply LibreNMS/OS/Gigamon.php:22]: Waiting for confirmation as the initial code I based this on is basically exactly his. Once I hear from him I will change this and the test files.

**Comment by hanserasmus** [comment]
@TheGreatDoc fixed with updated data.

**Comment by hanserasmus** [comment]
I have a few issues in this branch. Deleting it and will submit it again with less issues (hopefully).

**Pull request closed without merging**